### PR TITLE
New version: SeventhString.Transcribe version 9.30.2

### PR DESCRIPTION
**Pull request by vyryls**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/117638)

### CODE REVIEW

**Comment by vyryls** [comment]
@microsoft-github-policy-service agree

File: manifests/s/SeventhString/Transcribe/9.30.2/SeventhString.Transcribe.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 9.30.2
```
Comment (Trenly) [inline manifests/s/SeventhString/Transcribe/9.30.2/SeventhString.Transcribe.installer.yaml:5]:
```suggestion
PackageVersion: 9.30
```
This needs to match control panel, or there will be an upgrade loop.

File: manifests/s/SeventhString/Transcribe/9.30.2/SeventhString.Transcribe.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 9.30.2
```
Comment (Trenly) [inline manifests/s/SeventhString/Transcribe/9.30.2/SeventhString.Transcribe.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 9.30
```
This needs to match control panel, or there will be an upgrade loop.

File: manifests/s/SeventhString/Transcribe/9.30.2/SeventhString.Transcribe.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: SeventhString.Transcribe
+PackageVersion: 9.30.2
```
Comment (Trenly) [inline manifests/s/SeventhString/Transcribe/9.30.2/SeventhString.Transcribe.yaml:5]:
```suggestion
PackageVersion: 9.30
```
This needs to match control panel, or there will be an upgrade loop.

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**